commons/src/connections/threads.ts

```diff
@@ -10,6 +10,7 @@ import type {
   ConversationQuery,
   Conversation,
   MiddlewareResponse,
+  ThreadsWithSearchKeywordQuery,
```

Comment:
I love a verbose function and variable name, but I think this could be shortened to `SearchResultThreads`? WDYT?